src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1501,41 +1501,39 @@ internal enum ErrorCode
         ERR_NamedArgumentSpecificationBeforeFixedArgumentInDynamicInvocation = 8324,
         #endregion diagnostics introduced for C# 7.2
 
-        //PROTOTYPE(ReadonlyRefs): make err IDs contiguous before merging to master. 
-        //                         For now it is more convenient to have a gap to avoid conflicts with other added errors
-        ERR_RefConditionalAndAwait = 8401,
-        ERR_RefConditionalNeedsTwoRefs = 8402,
-        ERR_RefConditionalDifferentTypes = 8403,
-        ERR_BadParameterModifiers = 8404,
+        ERR_RefConditionalAndAwait = 8325,
```

Comment:
>ERR_RefConditionalAndAwait [](start = 8, length = 26)

Could you please put a region around these with a comment describing the feature and release they are supporting?

Reply:
Did not notice the pattern. Will do.

---
In reply to: [138723419](https://github.com/dotnet/roslyn/pull/21982#discussion_r138723419) [](ancestors = 138723419)